plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -1878,11 +1878,9 @@ fu_genesys_usbhub_device_prepare(FuDevice *device,
 		return TRUE;
 
 	fu_progress_set_id(progress, G_STRLOC);
-	fu_progress_add_step(progress, FWUPD_STATUS_SCHEDULING, 5, "detach");
-	for (guint i = 0; i < fw_types->len; i++) {
-		fu_progress_add_step(progress, FWUPD_STATUS_SCHEDULING, 100, NULL);
-		fu_progress_add_step(progress, FWUPD_STATUS_SCHEDULING, 100, NULL);
-	}
+	fu_progress_add_step(progress, FWUPD_STATUS_LOADING, 5, "detach");
+	for (guint i = 0; i < fw_types->len; i++)
+		fu_progress_add_step(progress, FWUPD_STATUS_LOADING, 100, NULL);
```

Comment:
Not convinced about this one -- the original code looks correct to me...

Reply:
Not with our definition of scheduling. It's meant to say it's running offline.